src/pages/ProfilePage.js

```diff
@@ -26,4 +26,10 @@ export const navPaths = [
     component: ProfilePage,
     title: 'Profile',
   },
+  {
+    name: 'oauth-callback',
+    path: '/oauth-callback',
```

Comment:
The callback url path is `/oauth_callback`: https://github.com/organizations/terra-bio/settings/apps/terra-app-dev

Reply:
Does this need to be `oauth_callback` here too?